docs/book/v3/helpers/json.md

```diff
@@ -1,21 +1,16 @@
 # Json
 
-When creating views that return JSON, it's important to also set the appropriate
-response header.  The JSON view helper does exactly that. In addition, by
-default, it disables layouts (if currently enabled), as layouts generally aren't
-used with JSON responses.
+This view helper simply encodes its argument to JSON.
 
-The JSON helper sets the following header:
+## Basic Usage
 
-```http
-Content-Type: application/json
+```php
+<?= $this->json(['example' => 'payload']) ?>
 ```
 
-Most XmlHttpRequest libraries look for this header when parsing responses to
-determine how to handle the content.
-
-## Basic Usage
+The helper accepts a second argument `$jsonOptions`, an associative array with one possible key: `prettyPrint`.
```

Comment:
If there is only one possible key, why is an array needed here?

Reply:
I know. Originally I made this a `bool` but re-considered. If anyone is using this helper, it'll be annoying to break BC for aesthetics. Additionally, it occurred to me that users might want to escape output for HTML, at that point, we can add an `escape` key or any others. That said, I'd personally prefer a `bool` but I didn't think it was worth the BC break. I'll happily change it if we all aggree it's stupid to require an obtuse array for a single flag! I'm personally in favour of the bool.